server/XunitTests/Repository/RecipeRepositoryTests.cs

```diff
@@ -0,0 +1,67 @@
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using CookBook.ExternalApi;
+using CookBook.Options;
+using FluentAssertions;
+using Moq;
+using Moq.Protected;
+using Xunit;
+
+public class RecipeRepositoryTests
+{
+    private Mock<HttpClientHandler> _handlerMock;
+    private ApiOptions _options;
+
+    public RecipeRepositoryTests()
+    {
+        _handlerMock = new Mock<HttpClientHandler>();
+        _handlerMock
+           .Protected()
+           // Setup the PROTECTED method to mock
+           .Setup<Task<HttpResponseMessage>>(
+              "SendAsync",
+              ItExpr.IsAny<HttpRequestMessage>(),
+              ItExpr.IsAny<CancellationToken>()
+           )
+           // prepare the expected response of the mocked http call
+           .ReturnsAsync(new HttpResponseMessage()
+           {
+               StatusCode = HttpStatusCode.OK
+           })
+           .Verifiable();
+
+
+        _options = new ApiOptions();
+        _options.ApiKey = "123";
+        _options.Server = "https://serverTest.com";
+    }
+
+    [Fact]
+    public void GetRecipePriceBreakdown()
+    {
+        var httpClient = new HttpClient(_handlerMock.Object);
+        var subjectUnderTest = new RecipeRepository(_options, httpClient);
+        var result = subjectUnderTest.GetRecipePriceBreakdown(12);
+
+        // ASSERT
```

Comment:
If you add AAA comments, add them always.

Reply:
dodam, jak bede kontynuował pozostale testy